rpm/gratia-probe.spec

```diff
@@ -449,6 +456,12 @@ The HTCondor-CE probe for the Gratia OSG accounting system
 %post htcondor-ce
 %customize_probeconfig -d htcondor-ce
 
+# Fix directory ownership if users have run condor_meter as root (SOFTWARE-5531)
+condor_ids=$(python3 -c 'import gratia.common.condor as condor; \
+             print(":".join([str(x) for x in core.get_condor_ids("htcondor-ce")]))')
+for dirname in $(awk -F '=' '/Folder=/ {print $2}' /etc/gratia/htcondor-ce/ProbeConfig | tr -d \"); do
```

Comment:
This is terrifying. If someone messes up their ProbeConfig then the next time Gratia gets upgraded it will recursively chown whatever they put in there, which could be very destructive.

You should chown the default directories only (and declare ownership of them in the %files list if you haven't already); if the admin decides to move them elsewhere, that's their problem.